nixos/modules/services/networking/zeronsd.nix

```diff
@@ -0,0 +1,119 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.zeronsd;
+  settingsFormat = pkgs.formats.json {};
+  inherit (lib) mkOption;
+in {
+  options.services.zeronsd.enable = lib.mkEnableOption (lib.mdDoc "ZeroNSD");
+
+  options.services.zeronsd.servedNetworks = mkOption {
```

Comment:
Is there no need to set the settings per network? I would have expected something like `services.zeronsd.networks` which is an attrset of network names and settings. This would also remove the need for a `enable` option as you can just check if it's `{}` in `mkIf`